src/components/sync-connected-sites.tsx

```diff
@@ -208,9 +209,33 @@ const SyncConnectedSiteControls = ( {
 						type={ syncDialogType }
 						localSite={ selectedSite }
 						remoteSite={ connectedSite }
-						onSubmit={ () => {
+						onSubmit={ ( tree ) => {
```

Comment:
Could we extract this as a separate function? I think it would make it easier to read and understand. Maybe calling it `getSyncOptionsFromTree` or something similar.
What do you think?